packages/core/package.json

```diff
@@ -114,6 +115,11 @@
     ]
   },
   "theiaExtensions": [
+    {
+      "entryPoints": {
```

Comment:
Why not just make this an array: I don't see the need to have an id for each entry point.